MySQL/node.sqlBuilder.js

```diff
@@ -0,0 +1,25 @@
+let logger = global.logger;
+
+module.exports = async (connectorData, inputData) => {
+	logger.trace("MySQL SQL Builder: Invoked!");
+	try {
+		logger.trace(`MySQL SQL Builder: input - ${JSON.stringify(inputData)}`);
+		if (!inputData.query) {
+			logger.error(`MySQL SQL Builder: SQL Query is required.`);
+			throw { message: "SQL Query is required." };
+		}
+		const values = inputData.values ?? [];
+
+		const [result, _] = await connectorData.client.execute(inputData.query, values);
```

Comment:
Don't assign anything to `_`. By default it is the variable for lodash.

Reply:
Fixed.